src/routes/cspReport.js

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+// In this router we implement the service to handle csp reports.
+
+import Router from '@koa/router';
+import body from 'koa-json-body';
+
+import { getLogger } from '../log';
+import { BadRequestError } from '../utils/errors';
+
+export function cspReportRoutes() {
```

Comment:
We should probably add some documentation for each route. It's a fairly "public" API, and would probably useful in the long term.

Reply:
There's actually a comment above in the same file. Do you think it should be here instead?

Reply:
I moved the comment.